### ISSUE TITLE
Could you share more details about the training process?

### ISSUE THREAD

**Issue by leenpluto** [Ucas-HaoranWei/Vary-tiny-600k#9]
I'm conducting training on 3*H200 GPUs and observing that the loss is converging quite slowly. The metrics are as follows:

Epoch 0: loss 2.1
Epoch 1: loss 2.03
Epoch 2: loss 1.97
Epoch 3: loss 1.86
I'm currently using the default configuration from the repository(batchszie 4,lr 0.0001). Would you recommend increasing the batch size and learning rate to improve convergence?
